dashboard/app/models/code_review_note.rb

```diff
@@ -29,9 +29,11 @@ def summarize
     {
       id: id,
       commenterName: commenter&.name,
+      commenterId: commenter&.id,
       comment: comment,
       isResolved: is_resolved,
-      createdAt: created_at
+      createdAt: created_at,
+      isFromTeacher: code_review.owner.memoized_teachers.include?(commenter)
```

Comment:
Does this end up requiring a separate query per note to get the owner's teachers?

Reply:
I'm wondering if we should calculate this when the note is created and just store it on the table.

Reply:
Nope, this was tested as part of the query optimization, because there is only one code review note owner and by using the `memoized_teachers` method, only one query is made to the database to fetch this value